x-pack/plugins/alerting/server/routes/maintenance_window/apis/find/transforms/transform_find_maintenance_window_to_response/v1.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformMaintenanceWindowToResponseV1 } from '../../../../transforms';
+import type { FindMaintenanceWindowsResponseV1 } from '../../../../../../../common/routes/maintenance_window/apis/find';
+import type { MaintenanceWindow } from '../../../../../../application/maintenance_window/types';
+import type { FindMaintenanceWindowsResult } from '../../../../../../application/maintenance_window/methods/find/types';
+
+export const transformFindMaintenanceWindowResponse = (
+  result: FindMaintenanceWindowsResult
+): FindMaintenanceWindowsResponseV1 => {
+  return {
+    body: {
+      page: result.page,
+      per_page: result.perPage,
+      total: result.data.length,
```

Comment:
The `total` should not be the length of the returned data but the total of all MW. This information is being returned by the SO find API. See my other comment above.